rosboard/compression.py

```diff
@@ -37,26 +38,26 @@ def encode_jpeg(img):
             img = np.expand_dims(img, axis=2)
             if not img.flags['C_CONTIGUOUS']:
                 img = img.copy(order='C')
-            return simplejpeg.encode_jpeg(img, colorspace = "GRAY", quality = 50)
+            return simplejpeg.encode_jpeg(img, colorspace = "GRAY", quality = 100)
```

Comment:
Will all the quality be the same, and if so may make sense to create a variable at the top of the file?

Reply:
Good point, just made a variable at the top! @rohitnambiarOW